nilmtk/elecmeter.py

```diff
@@ -421,7 +421,7 @@ def load(self, **kwargs):
             print("ElecMeter.load")
             print(self)
 
-        if 'sample_period' in kwargs:
+        if 'sample_period' in kwargs and kwargs['resample'] != False:
```

Comment:
this will raise a `KeyError` if 'resample' is not in `kwargs`.  Might be better to do something like: `if 'sample_period' in kwargs and kwargs.get('resample') != False:`  (because `.get()` returns `None` if the key does not exist in the dict).  Although it might be even clearer to do:

```python
if 'sample_period' in kwargs:
    kwargs.setdefault('resample', True)
```

(`setdefault(key, value)` won't change the value of a key if the key already exists)